webservices/common/models/itemized.py

```diff
@@ -47,7 +43,7 @@ class ScheduleA(BaseItemized):
         )'''
     )
     contributor_name = db.Column('contbr_nm', db.String, doc=docs.CONTRIBUTOR_NAME)
-    contributor_prefix = db.Column('contbr_prefix', db.String)
+    contributor_name_text = db.Column(TSVECTOR)
```

Comment:
The items that were at the bottom in the auxiliary fields section before are things that we use internally but is not exposed to the user. The TSVECTOR fields are a kind of index for searching, they break up text into pieces and weight them. These fields return text matches quickly. The output looks like `'barber':1A 'joe':3A 'rick':2A`